RELEASE_NOTES.md

```diff
@@ -1,10 +1,38 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Infra Client 15.6
+
+## Updated Resources
+
+## apt_repository
+
+The `apt_repository` resource now property escapes repository URIs instead of quoting them. This prevents failures when using the `apt-file` command, which was unable to parse the quotes URIs. Thanks for reporting this [@Seb-Solon](https://github.com/Seb-Solon)
```

Comment:
Would this be one quote or two quotes mentioned in "unable to parse the quotes URIs"? That would affect where the apostrophe goes...

Reply:
I think I fixed this sentence up now